### PR TITLE
Changements for v0.0.2

### PR DESCRIPTION
**Pull request by orblazer**
Implement NuxtI18nLink component, improve detection of language and implement option for default language

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,7 +45,16 @@ For example:
 }
 ```
 
+### Options of module
+- `languages` : An list available languages
+- `defaultLanguage` : The default language ***⚠ Required: The value need defined in `languages`***
```
Comment (njam) [inline README.md:50]: How about making it optional? We could just use the first language as the default. That way we keep the configuration backwards compatible.

File: src/templates/plugin.js [hunk 2]
```diff
@@ -27,11 +28,12 @@ export default ({app, store}) => {
   })
   app.i18n = new VueI18n({
     locale: store.state['i18n'].language,
-    fallbackLocale: options.languages[0],
+    fallbackLocale: options.defaultLanguage || options.languages[0],
     messages: messages,
     silentTranslationWarn: true
   })
 
+  Vue.availableLanguages = Object.keys(messages)
```
Comment (njam) [inline src/templates/plugin.js:36]: What's the reason to get the *available languages* from the messages/phrases, and not from the module config (`options.languages`)? I think it'd be good to do it consistently across the module, and the latter seems a more robust source for that, wdyt?

**Review by njam — CHANGES_REQUESTED** [review]
@orblazer very sorry to only get back to you about this now...
Are you still interested in getting this merged?

@christopheschwyzer maybe you want to take a look too?

**Comment by orblazer** [comment]
Yex, i go apply the changes in next days

**Comment by orblazer** [comment]
Hello, i have remake my PR for the last version and with better code.

The PR is here : #16

**Pull request closed without merging**